packages/twenty-front/src/modules/views/components/ViewBarDetails.tsx

```diff
@@ -29,21 +29,23 @@ const StyledBar = styled.div`
   border-top: 1px solid ${({ theme }) => theme.border.color.light};
   display: flex;
   flex-direction: row;
-  min-height: 32px;
+  align-items: center;
   justify-content: space-between;
+  position: relative;
+  padding: ${({ theme }) => theme.spacing(1)};
```

Comment:
this is not matching the Figma, on the figma we only have a padding-top and padding-bottom, no left and right